README.md

```diff
@@ -56,6 +56,48 @@ To add properties that are tracked in every event, you can set properties for a
     userProperties.key = "value";
     amplitude.setUserProperties(userProperties);
 
+# User Property Operations #
+
+The SDK supports the operations set, setOnce, unset, and add on individual user properties. The operations are declared via a provided `Identify` interface. Multiple operations can be chained together in a single `Identify` object. The `Identify` object is then passed to the Amplitude client to send to the server. The results of the operations will be visible immediately in the dashboard, and take effect for events logged after.
+
+1. `set`: this sets the value of a user property.
+
+    ```javascript
+    var identify = new amplitude.Identify().set('city', 'Boston, MA').set('country', 'United States');
```

Comment:
I'm not sure want to use the built-in fields as examples here since they operate differently from custom properties. If you do end up keeping it, "city" should just be "Boston" (we have a separate "region" field for state).